apps/public-docsite-resources/package.json

```diff
@@ -42,8 +42,6 @@
     "@fluentui/theme-samples": "^8.4.17",
     "@fluentui/react-monaco-editor": "^1.3.17",
     "office-ui-fabric-core": "^11.0.0",
-    "react": "16.14.0",
```

Comment:
this is incorrect. `dependencies` should remain for the time being in place as is for both libraries and applications.  

For libraries it's self explanatory - consumers install library and needs all dependencies
For apps - imagine this would need to be run within docker container or similar

Reply:
It's kind of debatable for unpublished apps, which are either only intended to work within the repo, or are intended to be bundled and uploaded in a way that makes dependencies irrelevant (React is included in the website and storybook bundles). I'm not sure the docker container type scenario matters here since they're never actually run that way, and would pretty thoroughly not work already due to single version policy with dev deps + private scripts package.

Regardless, I'll revert this and let you figure it out later.